A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_AirportsMonitor.js

```diff
@@ -0,0 +1,194 @@
+                // in which case the display will not be cleared and redrawn
+                if (_Values.length === 4) {
+                    this.icao1 = _Values[0][2].substr(2).trim()
+                    this.icao2 = _Values[1][2].substr(2).trim()
+                    this.icao3 = _Values[2][2].substr(2).trim()
+                    this.icao4 = _Values[3][2].substr(2).trim()
+                    this.dist1 = Math.round(parseFloat(_Values[0][3]))
+                    this.dist2 = Math.round(parseFloat(_Values[1][3]))
+                    this.dist3 = Math.round(parseFloat(_Values[2][3]))
+                    this.dist4 = Math.round(parseFloat(_Values[3][3]))
+                    // values are jumpy! mitigated by slow update rate, as only about every 30th value is erroneous
+                    this.magvar = SimVar.GetSimVarValue("MAGVAR", "degree")
+                    this.brng1 = Math.round(parseFloat(_Values[0][4]) - this.magvar)
+                    this.brng2 = Math.round(parseFloat(_Values[1][4]) - this.magvar)
+                    this.brng3 = Math.round(parseFloat(_Values[2][4]) - this.magvar)
+                    this.brng4 = Math.round(parseFloat(_Values[3][4]) - this.magvar)
+                    this.ll1 = new LatLong(_Values[0][0], _Values[0][1])
+                    this.ll2 = new LatLong(_Values[1][0], _Values[1][1])
+                    this.ll3 = new LatLong(_Values[2][0], _Values[2][1])
+                    this.ll4 = new LatLong(_Values[3][0], _Values[3][1])
+                }
+            }.bind(this))
+
+            // logic from FlightPlanManager.js
+            this.gs = SimVar.GetSimVarValue("GPS GROUND SPEED", "knots")
+            if (this.gs < 50) {
+                this.gs = 50
+            }
+
+            this.total_delta_t = 0
+        }
+
+        let s_per_h = 3600
+        let s_per_d = s_per_h * 24
+
+        // ETA offset in seconds
+        var eta1_s = Math.round(this.dist1 / this.gs * s_per_h)
+        var eta2_s = Math.round(this.dist2 / this.gs * s_per_h)
+        var eta3_s = Math.round(this.dist3 / this.gs * s_per_h)
+        var eta4_s = Math.round(this.dist4 / this.gs * s_per_h)
+
+        // absolute ETA in seconds
+        var utc_s  = Math.floor(SimVar.GetGlobalVarValue("ZULU TIME", "seconds"))
+        var utc1_s = (utc_s + eta1_s) % s_per_d
+        var utc2_s = (utc_s + eta2_s) % s_per_d
+        var utc3_s = (utc_s + eta3_s) % s_per_d
+        var utc4_s = (utc_s + eta4_s) % s_per_d
+
+        // components for ETAs in HHMM format
+        var h1 = Math.floor(utc1_s / s_per_h) || 0
+        var h2 = Math.floor(utc2_s / s_per_h) || 0
+        var h3 = Math.floor(utc3_s / s_per_h) || 0
+        var h4 = Math.floor(utc4_s / s_per_h) || 0
+        var m1 = Math.floor(utc1_s % s_per_h / 60) || 0
+        var m2 = Math.floor(utc2_s % s_per_h / 60) || 0
+        var m3 = Math.floor(utc3_s % s_per_h / 60) || 0
+        var m4 = Math.floor(utc4_s % s_per_h / 60) || 0
```

Comment:
Maybe also consider making those arrays and indexing them instead

```suggestion
        const eta1_s = Math.round(this.dist1 / this.gs * s_per_h)
        const eta2_s = Math.round(this.dist2 / this.gs * s_per_h)
        const eta3_s = Math.round(this.dist3 / this.gs * s_per_h)
        const eta4_s = Math.round(this.dist4 / this.gs * s_per_h)

        // absolute ETA in seconds
        const utc_s  = Math.floor(SimVar.GetGlobalVarValue("ZULU TIME", "seconds"))
        const utc1_s = (utc_s + eta1_s) % s_per_d
        const utc2_s = (utc_s + eta2_s) % s_per_d
        const utc3_s = (utc_s + eta3_s) % s_per_d
        const utc4_s = (utc_s + eta4_s) % s_per_d

        // components for ETAs in HHMM format
        const h1 = Math.floor(utc1_s / s_per_h) || 0
        const h2 = Math.floor(utc2_s / s_per_h) || 0
        const h3 = Math.floor(utc3_s / s_per_h) || 0
        const h4 = Math.floor(utc4_s / s_per_h) || 0
        const m1 = Math.floor(utc1_s % s_per_h / 60) || 0
        const m2 = Math.floor(utc2_s % s_per_h / 60) || 0
        const m3 = Math.floor(utc3_s % s_per_h / 60) || 0
        const m4 = Math.floor(utc4_s % s_per_h / 60) || 0
```